src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -489,7 +487,59 @@ select game
                 tournamentSсheduleInfo);
         }
 
-        private void ValidateGameInRoundOnCreate(
+        private static void IsFreeDayGameValidation(bool GameValidationIsFreeDayGame, TournamentScheduleDto tournamentScheduleInfo, GameResultDto game)
```

Comment:
This method doesn't return value, so name shouldn't starts with 'Is'.
Use camel case for params.